### ISSUE TITLE
Slides icon missing in profile page

### ISSUE THREAD

**Issue by Jean85** [joindin/joindin-web2#826]
Today I updated the slides links in my talks, since I recently bought a proper domain for my site.

Two of my oldest talks had the slides icon, on the right; now that I've updated the links to each talk, none has the icon: https://joind.in/user/Jean85/talks

The icon is properly shown in the event schedule page, i.e.: https://joind.in/event/symfonyday-italy-2018/schedule/list

Maybe we would need to centralize that small piece of template?

**Comment by svpernova09** [comment]
Looks like we just need to add the icon to that view. Don't think we'd want to consolidate the views since they are different pages.

**Comment by dstockto** [comment]
The view has the icon - there's something else that's changing when slides that were in there are updated. Indeed I did it to myself on one.

This talk had a slide icon on my page of talks: 
https://joind.in/event/phpworld-2016/phone-calls-and-sms-from-php

I updated the talk (really just clicked the save button, didn't change anything in the forms). The link still shows on the link above, but notice there's slides on older talks, but not on that one here: 

https://joind.in/user/dstockto/talks

I suspect the issue is likely something with the API or database.
